packages/plugins/IconMenu/README.md

```diff
@@ -11,40 +11,70 @@ _Please note that the average user has never seen the client before and just wan
+
 > ⚠️ This plugin can't be configured with the `mapConfiguration`, but is configured during client construction. Currently, no way exists to add plugins to it after the build took place.
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
 
-Usage during client build:
+| fieldName | type | description |
+| - | - | - |
+| initiallyOpen  | string?   | Id of the plugin which should be open on start; only applicable if the device doesn't have a small display  |
+| menus | Array | Defines which plugins should be rendered as part of the icon menu. |
+| layoutTag | enum NineLayoutTag | Defines where the icon menu is rendered. Only use `NineLayoutTag.TOP_RIGHT` here for the time being. |
+
+Use the configuration during client build.
+
+Example Configuration:
+```js
+{
+  initiallyOpen: 'layerChooser',
+  displayComponent: true,
+  menus: [
+    {
+      plugin: PolarPluginLayerChooser({}),
+      icon: 'fa-layer-group',
+      id: 'layerChooser',
+    },
+    {
+      plugin: PolarPluginDraw({}),
+      icon: 'fa-pencil',
+      id: 'draw',
+      hint: 'Draw or write something on the map'
+    },
+  ],
+  layoutTag: NineLayoutTag.TOP_RIGHT,
+}
+```
+
+### iconMenu.menus
+
+| fieldName | type | description |
+| - | - | - |
+| plugin | polar plugin | The plugin that should be part of the icon menu |
+| icon | string? | Icon for icon menu button. If given, render a button with the icon. When clicked, open the content of the configured plugin. If not given, render the plugin content as is inside the IconMenu. Current examples for the usage without icon include Zoom and Fullscreen.  |
+| id | string | Id of the plugin, used to resolve hint locale. |
+| hint | string? | Overrides the default hint displayed for the icon menu button |
 
+Example Configuration:
 ```js
 {
-  // id of the plugin which should be open on start; only applicable if the device doesn't have a small display
-  initiallyOpen: 'some',
   menus: [
     {
-      // use plugin as child
-      plugin: SomePolarPlugin({}),
-      /*
-       * Icon for icon menu button
-       * If given, render a button with the icon. When clicked, open the content of the configured plugin.
-       * If not given, render the plugin content as is inside the IconMenu.
-       * Current examples for the usage without icon include Zoom and Fullscreen,
-       */
-      icon: 'fa-book-atlas',
-      // used to resolve hint locale
-      id: 'some',
-      // optionally override the hint displayed for the icon menu button
-      hint: 'Some Plugin',
+      plugin: PolarPluginLayerChooser({}),
+      icon: 'fa-layer-group',
+      id: 'layerChooser',
     },
     {
-      plugin: AnotherPolarPlugin({}),
-      id: 'another',
+      plugin: PolarPluginDraw({}),
+      icon: 'fa-pencil',
+      id: 'draw',
+      hint: 'Draw or write something on the map'
     },
   ],
 }
 ```
 
-### menus
+### modes
```

Comment:
```suggestion
## Modes
```

I like the rename. Just removed a # since it's not part of the configuration.

Reply:
f536941285d77b6098b3d622547a8a24d77d1db0